src/components/ScholarshipListCard.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  Box,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  actions: {
+    padding: theme.spacing(2),
+    paddingTop: theme.spacing(1),
+  },
+  content: {
+    padding: theme.spacing(3),
+  },
+  subtitle: {
+    fontWeight: '500',
+  },
+  deadline: {
+    fontWeight: 'bold',
+    marginBottom: theme.spacing(1),
+  },
+  description: {
+    display: '-webkit-box',
+    '-webkit-line-clamp': 5,
+    lineClamp: 5,
+    '-webkit-box-orient': 'vertical',
+    overflow: 'hidden',
+    whiteSpace: 'pre-line',
+    marginBottom: theme.spacing(3),
+  },
+  tagWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  tag: {
+    marginRight: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    borderRadius: '50px',
+    border: '1px solid #3FB1B5',
```

Comment:
We haven't updated the site theme to match the design yet but the outline color is going to be the same as the primary color. [Outlined chips](https://material-ui.com/components/chips/#outlined-chips) have an easy way to use the theme's primary color.